src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/MapCode/MapCodeEndpoint.cs

```diff
@@ -59,7 +59,7 @@ internal sealed class MapCodeEndpoint : IRazorDocumentlessRequestHandler<LSP.Map
     public bool MutatesSolutionState => false;
 
     public async Task<LSP.WorkspaceEdit?> HandleRequestAsync(
-        LSP.MapCodeParams request,
+        LSP.VSInternalMapCodeParams request,
```

Comment:
Nit: The LSP is presumably redundant here, give it wasn't necessary on line 40.

Presumably could remove the using for it entirely